static/styles/zulip.css

```diff
@@ -580,7 +580,7 @@ label {
 
 /* Bootstrap's focus outline uses -webkit-focus-ring-color which doesn't exist in Firefox */
 a:not(:active):focus,
-button:focus,
+button:not(.poll-option, .add-task, .poll-vote):focus,
```

Comment:
Can we avoid this by just setting the `outline` property specifically for the new buttons? I don't want to add a `not` here.

Reply:
Also removed another similar `not` block in `zulip.css` which defined outline color